http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/leonardoServiceContractModel.scala

```diff
@@ -231,3 +234,62 @@ object GetRuntimeResponse {
     diskConfig
   )
 }
+
+final case class CreateAppRequest(kubernetesRuntimeConfig: Option[KubernetesRuntimeConfig],
+                                  appType: AppType,
+                                  diskConfig: Option[PersistentDiskRequest],
```

Comment:
shouldn't this be a non optional?

Reply:
also would be nice to move these contract data types to core..similar to https://github.com/DataBiosphere/leonardo/blob/develop/core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/diskRoutesModels.scala...it's a lot more work to move them later